src/controllers/v1/translator.js

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+import dotenv from 'dotenv';
+import Response from '../../utils/response';
+
+dotenv.config();
+
+const translatorController = async (req, res) => {
+  let text = req.body.text || '';
+  const lang = req.body.lang;
+
+  if (text.length > 280) {
+    return Response.inputError(res, 'text should be no more than 280 characters');
+  }
+  text = encodeURIComponent(text);  
+
+  const apiUrlBase = 'https://translate.yandex.net/api/v1.5/tr.json/translate';
```

Comment:
lets keep apiUrlBase in a service or something similar, to make swtiching it out easier. If you have questions let me know

Reply:
I have now made the url an environmental variable